autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/AutofillEventListener.kt

```diff
@@ -51,12 +51,6 @@ interface AutofillEventListener {
      */
     fun onUseEmailProtectionPrivateAlias(originalUrl: String, duckAddress: String)
 
-    /**
-     * Called when user chooses not to autofill any duck address.
-     * @param originalUrl the URL of the page that prompted the user to use a duck address
-     */
-    fun onRejectToUseEmailProtection(originalUrl: String)
```

Comment:
redundant now; removing the API ([API proposal: removing AutofillEventListener.onRejectToUseEmailProtection](https://app.asana.com/0/1205570447955448/1205570447955448/f))